<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestSetup</class>
 <widget class="QDialog" name="TestSetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>1024</width>
     <height>511</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="IpEdit" name="serverIp">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>170</y>
       <width>221</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="MyLineEdit" name="siteCode">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>235</y>
       <width>221</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="maxLength">
      <number>6</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>170</y>
       <width>100</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Server IP</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="siteName">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>235</y>
       <width>100</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Site Code</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="serverSave">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>200</y>
       <width>120</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>저장</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QGroupBox" name="groupBox2">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>42</y>
       <width>431</width>
       <height>360</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string>LAN2</string>
     </property>
     <widget class="QRadioButton" name="lan2Dhcp">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>191</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>자동으로 IP주소 받기</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="lan2Static">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>191</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>다음 IP주소 사용</string>
      </property>
     </widget>
     <widget class="IpEdit" name="lan2IP">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>110</y>
        <width>201</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>125</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>IP 주소</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="IpEdit" name="lan2Subnetmask">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>162</y>
        <width>201</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>162</y>
        <width>125</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>서브넷마스크</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lan2Mac">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>267</y>
        <width>201</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>267</y>
        <width>125</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>MAC 주소</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>214</y>
        <width>125</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>게이트웨이 주소</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="IpEdit" name="lan2Gateway">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>214</y>
        <width>201</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QRadioButton" name="lan2NotUsed">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>191</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>사용안함</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="lan2DefaultGateway">
      <property name="geometry">
       <rect>
        <x>17</x>
        <y>312</y>
        <width>161</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>기본 게이트웨이</string>
      </property>
     </widget>
    </widget>
    <widget class="IpEdit" name="dnsPrimary">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>408</y>
       <width>201</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="IpEdit" name="dnsSecondary">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>461</y>
       <width>201</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="serialNoLabel">
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>470</y>
       <width>71</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>일련번호:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>408</y>
       <width>161</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>기본 DNS서버 주소</string>
     </property>
    </widget>
    <widget class="QLabel" name="serialNo">
     <property name="geometry">
      <rect>
       <x>900</x>
       <y>470</y>
       <width>111</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0000000000</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>461</y>
       <width>161</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>보조 DNS서버 주소</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>42</y>
       <width>431</width>
       <height>361</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string>LAN1</string>
     </property>
     <widget class="QRadioButton" name="lan1Dhcp">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>191</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>자동으로 IP주소 받기</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="lan1Static">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>191</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>다음 IP주소 사용</string>
      </property>
     </widget>
     <widget class="IpEdit" name="lan1IP">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>80</y>
        <width>201</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>125</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>IP 주소</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="IpEdit" name="lan1Subnetmask">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>132</y>
        <width>201</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>132</y>
        <width>125</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>서브넷마스크</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lan1Mac">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>237</y>
        <width>201</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>237</y>
        <width>125</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>MAC 주소</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>184</y>
        <width>125</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>게이트웨이 주소</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="IpEdit" name="lan1Gateway">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>184</y>
        <width>201</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QCheckBox" name="lan1DefaultGateway">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>310</y>
        <width>161</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>기본 게이트웨이</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="save_2">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>-1</y>
       <width>120</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>저장</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="server">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>4</y>
     <width>120</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>서버설정</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="network">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>4</y>
     <width>120</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>네트워크 설정</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>58</y>
     <width>1024</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="close">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>3</y>
     <width>120</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>닫기</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">mylineedit.h</header>
  </customwidget>
  <customwidget>
   <class>IpEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">ipedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>close</sender>
   <signal>clicked()</signal>
   <receiver>TestSetup</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>949</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
